src/CommandingOfficers/Commander.java

```diff
@@ -63,7 +63,7 @@
   private double myAbilityPower = 0;
 
   private ArrayList<CommanderAbility> myAbilities = null;
-  private String myActiveAbilityName = "";
+  private CommanderAbility myActiveAbility = null; // This should move up to Army later on.
```

Comment:
Should it? What if an army has multiple commander with currently-active abilities? I guess we'll figure that out when someone gets around to implementing armies.

Reply:
Removed the comment in PR2